pkg/integration/mock.go

```diff
@@ -72,17 +72,6 @@ func SetMockData(ctx context.Context, clientset kubernetes.Interface, namespace
 
 	secret, err := clientset.CoreV1().Secrets(namespace).Get(ctx, replicatedSecretName, metav1.GetOptions{})
 	if err != nil {
-		if kuberneteserrors.IsNotFound(err) {
-			data := map[string][]byte{
-				replicatedIntegrationMockDataKey: b,
-			}
-			err = createReplicatedSecret(ctx, clientset, namespace, data)
```

Comment:
just noticed this may require a unit test update if we don't want the secret being created anymore if it's not there.  seems ok since it's in the chart templates, so someone would probably have to delete it manually for this to happen